examples/inputs/example_01a/config.yml

```diff
@@ -18,26 +18,4 @@ example_01a:
       maximum_bid_price: 3000
       minimum_bid_price: -500
       price_unit: EUR/MWh
-      market_mechanism: pay_as_clear
-
-day_ahead_market:
-  start_date: 2019-01-01 00:00
-  end_date: 2019-04-01 00:00
-  time_step: 1h
-  save_frequency_hours: 24
-  markets_config:
-    EOM:
-      operator: EOM_operator
-      product_type: energy
-      products:
-        - duration: 1h
-          count: 24
-          first_delivery: 1h
-      opening_frequency: 24h
-      opening_duration: 24h
-      volume_unit: MWh
-      maximum_bid_volume: 100000
-      maximum_bid_price: 3000
-      minimum_bid_price: -500
-      price_unit: EUR/MWh
-      market_mechanism: pay_as_clear
+      market_mechanism: pay_as_clear
```

Comment:
I would like to keep the day_ahead_market..? or is there any reason to remove it?